src/styles/main.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import "./blocks/variables/variables";
```

Comment:
There is no reset or normalization of CSS properties. It's a good practice to include a CSS reset or normalization to ensure cross-browser compatibility and a clean slate for your styles.